docs/guides/remove-edit-buttons.rst

```diff
@@ -0,0 +1,25 @@
+Removing "Edit on ..." Buttons from Documentation
+=================================================
+
+When building your documentation, Read the Docs automatically adds buttons at
+the top of your documentation that point readers to your repository to make
+changes. For instance, if your repository is on GitHub, a button that says "Edit
+on GitHub" is added to your documentation to make it easy for readers to author
+new changes.
+
+The only way to remove these links currently is to override the Read the Docs
+theme templates:
+
+* In your Sphinx project path, add a path called ``_templates``. If you use a
```

Comment:
"add a path" sounds weird to me. I'd say that I prefer "create a folder" or "make a directory" or similar.